### PR TITLE
Add godot3 redirect

### PR DESCRIPTION
**Pull request by Frostbyte4664**
The Ubuntu package for Godot and its executable are both called godot3. This pull request adds a redirect profile and enables it in firecfg.

### CODE REVIEW

**Review by rusty-snake — APPROVED** [review]

File: godot3.profile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# Description: multi-platform 2D and 3D game engine with a feature-rich editor
+# This file is overwritten after every install/update
+# Persistent local customizations
+include godot3.local
+# Persistent global definitions
+# added by included profile
+#include globals.local
+
+# Redirect
+include godot
```
Comment (rusty-snake) [inline godot3.profile:11]:
```suggestion
include godot.profile
```

**Review by rusty-snake — CHANGES_REQUESTED** [review]

**Comment by rusty-snake** [comment]
Screenshot

I'm a time traveler

**Review by rusty-snake — APPROVED** [review]

**Review by kmk3 — CHANGES_REQUESTED** [review]
godot3.profile should be in the same directory as godot.profile.

**Review by kmk3 — APPROVED** [review]
LGTM.

**Comment by kmk3** [comment]
Merged, thanks!

**Pull request merged**